include/itkCuberilleImageToMeshFilter.hxx

```diff
@@ -29,16 +29,17 @@ namespace itk
 
 template<class TInputImage, class TOutputMesh, class TInterpolator>
 CuberilleImageToMeshFilter<TInputImage,TOutputMesh,TInterpolator>
-::CuberilleImageToMeshFilter()
+::CuberilleImageToMeshFilter() :
+  m_IsoSurfaceValue( NumericTraits< InputPixelType >::One ),
+  m_MaxSpacing( NumericTraits< SpacingValueType >::One ),
```

Comment:
This member variable was not being initialized. Note that I used ::One (instead of ::max() for example) to initialize the member variable. Please check if this is acceptable/correct.